frontend/src/components/Cart/CartSidebarItem.vue

```diff
@@ -82,6 +102,39 @@ export default {
 			this.$el.scrollIntoView({behavior: "instant", block: "nearest"});
 		},
 
+		handle_pointer_start(e) {
+			this.startX = e.touches[0].clientX;
+			this.currentX = this.startX;
+		},
+
+		handle_pointer_move(e) {
+			this.currentX = e.touches[0].clientX;
+			const deltaX = this.currentX - this.startX;
+			// Set Max Enough to trigger touch
+			const threshold = 15;
+
+			if (deltaX < threshold) {
+				this.$refs.cart_item.style.marginLeft = `${Math.min(Math.max(deltaX, -55), 0)}px`;
```

Comment:
do not directly mutate style. Use a data variable and bind it to the div's :style